web/crux/src/app/health/health.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable, Logger } from '@nestjs/common'
+import { HealthResponse, ServiceStatus } from 'src/grpc/protobuf/proto/crux'
+import { PrismaService } from 'src/services/prisma.service'
+
+@Injectable()
+export class HealthService {
+  private logger = new Logger(HealthService.name)
+
+  constructor(private prisma: PrismaService) {}
+
+  async getHealth(): Promise<HealthResponse> {
+    let lastMigration: string = null
+    try {
+      lastMigration = await this.prisma.findLastMigration()
+    } catch (err) {
+      const error: Error = err
```

Comment:
Why do you have to create an error object?

Reply:
cause typescript is a very fine language but apparently you can't give a type to a variable in the catch statement